internal/preflight/preflight.go

```diff
@@ -131,15 +131,23 @@ func (p *preflightHelper) verifyImage(ctx context.Context, mapping *kmmv1beta1.K
 	return false, fmt.Sprintf("image %s does not contain kernel module for kernel %s on any layer", image, kernelVersion)
 }
 
-func (p *preflightHelper) verifyBuild(ctx context.Context, mapping *kmmv1beta1.KernelMapping, mod *kmmv1beta1.Module, kernelVersion string) (bool, string) {
+func (p *preflightHelper) verifyBuild(ctx context.Context,
+	pv *kmmv1beta1.PreflightValidation,
```

Comment:
How about passing just the boolean instead?

Reply:
right now i prefer to pass the pointer to the whole object, since i don't know yet which data i will need in order to implement the sign section. Anyway, once Chris commit is in and we know for sure the how it is going to be implemented, i will probably submit addition PR for a better refactoring